### ISSUE TITLE
Navbar: navbar `<nav>` should have an `aria-label`

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#2212]
Navbar needs to have an optional `aria-label` config.

Reason being is that if there is more than one `<nav>` component on the page then we want screen reader to announced that the navbar is 'main menu' or something like that.
